.github/workflows/docker-build.yaml

```
@@ -38,17 +38,24 @@ jobs:
         run: find ${{ matrix.BUILD_NAME }}-out -type f | xargs sha256sum
       - name: 'Record the git commit and any tags'
         run: git log | head -n1 > ${{ matrix.BUILD_NAME }}-out/commit.txt
-      - name: 'Upload ${{ matrix.BUILD_NAME }} wasm module'
+      - name: 'Upload ${{ matrix.BUILD_NAME }} nns-dapp wasm module'
         uses: actions/upload-artifact@v3
         with:
-          name: NNS ${{ matrix.BUILD_NAME }} wasm module and arguments
+          name: nns-dapp for ${{ matrix.BUILD_NAME }}
           path: |
             ${{ matrix.BUILD_NAME }}-out/commit.txt
             ${{ matrix.BUILD_NAME }}-out/nns-dapp.wasm
             ${{ matrix.BUILD_NAME }}-out/nns-dapp-arg-${{ matrix.DFX_NETWORK }}.did
             ${{ matrix.BUILD_NAME }}-out/nns-dapp-arg-${{ matrix.DFX_NETWORK }}.bin
             ${{ matrix.BUILD_NAME }}-out/frontend-config.sh
             ${{ matrix.BUILD_NAME }}-out/deployment-config.json
+      - name: 'Upload ${{ matrix.BUILD_NAME }} sns_aggregator'
```

A suggestion for consistency because of name: 'Upload ${{ matrix.BUILD_NAME }} nns-dapp `wasm module'`

```suggestion
      - name: 'Upload ${{ matrix.BUILD_NAME }} sns_aggregator wasm module'
```